tenacity/before_sleep.py

```diff
@@ -32,6 +32,7 @@ def before_sleep_log(
     logger: "logging.Logger",
     log_level: int,
     exc_info: bool = False,
+    sec_format: str = "%0.3f",
```

Comment:
```suggestion
    sec_format: str = "%0.gf",
```
WDYT?

Reply:
Yep, that's better as it doesn't require changing the test cases either. I will repush.